state-chain/pallets/cf-vaults/src/tests.rs

```diff
@@ -68,3 +68,19 @@ fn vault_start_block_number_is_set_correctly() {
 		assert_last_event!(crate::Event::VaultActivationCompleted);
 	});
 }
+
+#[test]
+fn vault_start_block_number_not_set_when_chain_not_initialized() {
```

Comment:
Is this test testing the change? I reverted the change and run this test, it still passes?